### ISSUE TITLE
Ignore rubocop inline directives when calculating LineLength

### ISSUE THREAD

**Issue by scottmatthewman** [rubocop/rubocop#774]
Hi,

Just started using Rubocop, so apologies first if I'm missing something or am duplicating an issue (I did look, but you never know).

I've got a method where one line is using multiple assignment to quickly get the requisite parts of a regex match. Due to the parenthesis rules in the regex, one of the assignments is unneeded, and is named as such, e.g. (simplified):

``` ruby
discarded, a, b, c = source.match(regex).to_a
```

Rubocop is doing its job by flagging up a UselessAssignment error, so I tried adding a one-line disable directive:

``` ruby
discarded, a, b, c = source.match(regex).to_a # rubocop:disable UselessAssignment
```

That disables the UselessAssignment check for that line, which is what I want – but the addition of the inline comment now triggers a LineLength failure...

Would it be possible to ensure that LineLength ignores these sort of directives in its calculations? I'm not familiar enough with Rubocop's internals (yet) to suggest what, though…
